addons/binding/pom.xml

```diff
@@ -34,6 +34,7 @@
     <module>org.openhab.binding.feed</module>
     <module>org.openhab.binding.feed.test</module>
     <module>org.openhab.binding.ftpupload</module>
+    <module>org.openhab.binding.hdpowerview</module>
```

Comment:
This line should be deleted. `hdpowerview` is already migrated. Also remove the next line `groheondus` is being moved.